docs/src/content/docs/utilities/has-integration.mdx

```diff
@@ -26,25 +26,105 @@ import { Tabs, TabItem } from '@astrojs/starlight/components';
 	```
 	</TabItem>
 	<TabItem label="Standalone utilities">
-	```ts title="integration/index.ts" "hasIntegration"
+	```ts title="my-integration/index.ts" "hasIntegration"
+	import type { AstroIntegration } from "astro";
+	import { hasIntegration } from "astro-integration-kit";
+
+	export default function myIntegration(): AstroIntegration {
+		return {
+			name: "my-integration",
+			hooks: {
+				"astro:config:setup": ({ config, logger }) => {
+					if (hasIntegration({
+						name: "@astrojs/tailwind",
+						config,
+					})) {
+						logger.info("Tailwind is installed!")
+					}
+				},
+			}
+		}
+	}
+	``` 
+	</TabItem>
+</Tabs>
+
+### Relative position check
+
+Sometimes two integrations must be installed in an specific order to work together correctly.
+
+For that use-case, this utility accepts optional `position` and `relativeTo` parameters to check for the presence of one integration in relation to another.
+
+Checking for the presence of an integration in relation to an uninstalled integration will result in an error.
+
+<Tabs>
+	<TabItem label="Extended hooks">
+
+	With extended hooks, `hasIntegration` can receive the position and relative reference as optional positional parameters.
+	If a relative reference is not given the result will be relative to the current integration being defined.
+
+	```ts title="my-integration/index.ts" ins=/"(?:before|after)"(?:, ".*"\\))?/
```

Comment:
I'm not convinced by the result here (eg. the ending parenthesis is highlighted but not the first). Could it be possible to highlight the whole `hasIntegration(...)` thing?

Reply:
> the ending parenthesis is highlighted but not the first

Oh that was a mistake, it should've been a positive lookahead.

> Could it be possible to highlight the whole hasIntegration(...) thing?

Surely it is, but then I'd do it with the neutral highlight like the first example in the page, not as insert since the change is just the parameters